src/analysis/conversion_type.rs

```diff
@@ -1,11 +1,18 @@
-use crate::{config::gobjects::GObject, env, library::*};
+use crate::{env, library::*};
 
-#[derive(Clone, Copy, Debug, Eq, PartialEq)]
+use std::sync::Arc;
+
+#[derive(Clone, Debug, Eq, PartialEq)]
 pub enum ConversionType {
-    Direct,  //coded without conversion
-    Scalar,  //coded with from_glib
-    Pointer, //coded with from_glib_xxx
-    Borrow,  //same as Pointer, except that use from_glib_borrow instead from_glib_none
+    Direct,           // coded without conversion
```

Comment:
At this point, we could turn those comments into doc comments I think. That'd also make the formatting easier.

Reply:
Sure